lexicons/com/atproto/repo/applyWrites.json

```diff
@@ -37,6 +37,16 @@
           }
         }
       },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "commitCid": { "type": "string", "format": "cid" },
+            "commitRev": { "type": "string" }
```

Comment:
If we want these to be required when either is present we could reshape it as `commit: { cid, rev }`.

Reply:
yeah i was back and forth on this, but i dig 👍